### ISSUE TITLE
Fixed inputs to packed ops should be uploaded as packed

### ISSUE THREAD

**Issue by annxingyuan** [tensorflow/tfjs#804]
Currently, the fixed inputs to packed ops such as batchnormalization must be packed every time the kernel runs. They should be uploaded as packed in the first place.
